File: submissions/sm_014_kondreddy/week_03/day_02/session_02/assignment/taxcalculator.js
var amount, tax, savings, rebate;
amount = 700000;
savings = 200000; 
if (amount < 250000) {
    tax = 0;
    console.log('tax:', tax)
}
else if (amount >= 250000 && amount < 500000) {
    tax = 250000*0.1;
    console.log('tax:',tax)
}
else if (amount >= 500000 && amount < 1000000) {
    tax = 500000*0.2;
    console.log('tax:',tax)
}
else if (amount >= 1000000) {
    tax = 200000*0.3;
    console.log('tax:',tax)
}
 if(amount < 500000){
rebate = amount-savings*0.5;
console.log('rebate:',rebate)
}
else if (amount < 1000000) {
rebate = amount-savings*0.3;
console.log("rebate:", rebate)
}
else if (amount >= 1000000) {
    rebate = amount-savings*0.1;
    if (rebate > 50000) {
        console.log("rebate:", 50000)
    }
    else {
    console.log("rebate:", rebate)
    }
}

File: submissions/sm_012_gaurav/week_13/day_5/session_1/tax_calculator.py
income = float(input("Enter Annual Income: "))
savings = float(input("Enter Savings: "))
if savings > income:
    print("Savings cannot be greater than Income.")
else:
    if income <= 500000:
        rebate = min(50000, 0.5*savings)
    elif income > 500000 and income <= 1000000:
        rebate = min(50000, 0.3*savings)
    else:
        rebate = min(50000, 0.1*savings)
    income -= rebate
    if income <= 250000:
        tax = 0
    elif income > 250000 and income <= 500000:
        tax = 0.1*(income - 250000)
    elif income > 500000 and income <= 1000000:
        tax = 0.1*250000 + 0.2*(income - 500000)
    else:
        tax = 0.1*250000 + 0.2*500000 + 0.3*(income - 1000000)
    print("Tax is", tax)